lib/reply.js

```diff
@@ -104,7 +106,16 @@ Object.defineProperties(Reply.prototype, {
   }
 })
 
+Reply.prototype.hijack = function () {
+  this[kReplyHijacked] = true
+  return this
+}
+
 Reply.prototype.send = function (payload) {
+  if (this[kReplyHijacked] === true) {
+    return this
+  }
+
   if (this[kReplyIsRunningOnErrorHook] === true) {
     throw new FST_ERR_SEND_INSIDE_ONERR()
   }
```

Comment:
I feel like these bits are core enough to Fastify that they should probably be tested, but my guess is you are waiting for feedback on the approach before doing a bunch of that work, 👍

Reply:
Yes exactly I was waiting for some feedback on the approach I implemented before adding those